CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# How to Contribute!
+
+Thanks for helping out!
+
+## Reporting Issues
+The best way to [report an issue is through Github](../../issues). The owners of the repo should get an email notification whenever a new issue is created.
+
+
+## Development 
+This section details the steps to setup the project for development.
+
+### Environment Setup and Tools
+Cloning this repo requires the use of Git, or you can use the template feature provided by Github. You can also download an archive of the repository contents using the Github "Download" link and extract this to access all of the files and create a new Git repository with them.
+
+Once you have a framework and development environment chosen for your project you should update your repo with specifics about how to install the tools and dependencies needed to run/debug/develop the application.
```

Comment:
Add Setup/Running steps for development

Reply:
see https://github.com/devlinjunker/template.hapi.rest/pull/49/files#diff-6a3371457528722a734f3c51d9238c13R15